cluster/sync.sh

```diff
@@ -35,11 +36,16 @@ for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
     ./cluster/cli.sh ssh "node$(printf "%02d" ${i})" "sudo sysctl -w user.max_user_namespaces=1024"
 done
 
-# Install CDI
-./cluster/kubectl.sh apply -f ./manifests/generated/cdi-operator.yaml
-./cluster/kubectl.sh apply -f ./manifests/generated/cdi-operator-cr.yaml
+
```

Comment:
Is this not installing CDI?

Reply:
The manifests location has changed from manifests/generated to _out/manifests/release

Reply:
Final manifests are moved to _out/manifests/release/